WORKSPACE

```diff
@@ -56,7 +56,7 @@ git_repository(
 http_file(
     name = "xctestrunner",
     executable = 1,
-    urls = ["https://github.com/google/xctestrunner/releases/download/0.2.5/ios_test_runner.par"],
+    urls = ["https://github.com/material-foundation/xctestrunner/releases/download/0.2.9/ios_test_runner.par"],
```

Comment:
Please add a TODO and an Issue to restore this back to the non-forked version in the future.

Reply:
Addressed in d316ff4700b2ae66e312e35f6b8f8038eb12e148